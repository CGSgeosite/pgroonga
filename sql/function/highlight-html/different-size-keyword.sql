SELECT pgroonga.highlight_html(
  '100㍉メートル',
  ARRAY['ミリ']);
