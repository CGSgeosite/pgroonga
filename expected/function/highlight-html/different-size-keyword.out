SELECT pgroonga.highlight_html(
  '100㍉メートル',
  ARRAY['ミリ']);
               highlight_html               
--------------------------------------------
 100<span class="keyword">㍉</span>メートル
(1 row)

